content/blog/2023/when-to-choose-a-responsive-web-app.md

```diff
@@ -0,0 +1,160 @@
+---
+title: When to Choose a Responsive Web App
+sub: Part 1 - Responsive Web App vs PWA vs Native Mobile App
+author: sondra
+date: 2023-05-05
+tags:
+  - Article
+  - Business Development
+  - Push Notifications
+  - Recommendations
+  - Startups
+  - User Experience
+image:
+  src: blog/2023/part1-web.jpg
+  alt: |
+    spider web with dew drops
+summary: |
```

Comment:
There's a lot of duplication between the summary and the opening paragraph. Since we show both in order on the page, that stands out.